env/production/api_gateway/terragrunt.hcl

```diff
@@ -0,0 +1,22 @@
+include {
+  path = find_in_parent_folders()
+}
+
+inputs = {
+  api-description   = "Terraform Serverless api for COVID Alert metrics collection"
+  apiKeyName        = "in-app-metrics-key"
+  api_gateway_rate  = "10000"
+  api_gateway_burst = "5000"
+  billing_tag_key   = "CostCentre"
+  billing_tag_value = "CovidShield"
+}
+
+terraform {
+  source = "git::https://github.com/cds-snc/covid-alert-metrics-terraform//aws/api_gateway?ref=v${get_env("INFRASTRUCTURE_VERSION")}"
+  extra_arguments "extra_args" {
+    commands = "${get_terraform_commands_that_need_vars()}"
+    optional_var_files = [
+      "${find_in_parent_folders("variables.auto.tfvars", "ignore")}",
```

Comment:
Question about loading `variables.auto.tfvars`.  Instead of autoloading this file, could we set this using the environment's `env_vars.hcl` and the root [`terragrunt.hcl`](https://github.com/cds-snc/covid-alert-metrics-terraform/blob/main/env/terragrunt.hcl#L5-L10)?

Reply:
Instead of using variables.auto.tfvars we should just be configuring this directly in the terragrunt.hcl as inputs on the module. 

This way we can be more explicit in what we are setting and we don't need to look up configuration in the auto.tfvars file.